Error in <TFile::ReadBuffer>: error reading all requested bytes from file mc_Simulation_run0_0_Test_1evt_pi+_10_10.root, got 286 of 300
Error in <TFile::Init>: mc_Simulation_run0_0_Test_1evt_pi+_10_10.root failed to read the file type data.
Traceback (most recent call last):
  File "/lustre/work/samumcki/dSiPM_NN/tensorMaker.py", line 181, in <module>
    main()
  File "/lustre/work/samumcki/dSiPM_NN/tensorMaker.py", line 99, in main
    input_file = ROOT.TFile(input_file_path, "READ")
                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/samumcki/miniconda3/lib/python3.12/site-packages/ROOT/_pythonization/_tfile.py", line 88, in _TFileConstructor
    raise OSError('Failed to open file {}'.format(args[0]))
OSError: Failed to open file mc_Simulation_run0_0_Test_1evt_pi+_10_10.root
mv: cannot stat 'tensor_0_0_pi+_10/npy/*': No such file or directory
